src/modal/modal-config.ts

```diff
@@ -69,8 +69,9 @@ export interface NgbModalOptions {
 
   /**
    * Size of a new modal window.
+   * 'sm' | 'md' | 'lg' | 'xl' | 'custom-class-name'
    */
-  size?: 'sm' | 'lg' | 'xl';
+  size?: string;
```

Comment:
Couldn't we keep `'sm' | 'md' | 'lg' | 'xl' | string` here ?

@maxokorokov don't we have something similar somewhere already ?

Reply:
2c, the `md` would have no effect, it is implicit one and there is no class for that size:
https://getbootstrap.com/docs/4.4/components/modal/#optional-sizes